### PR TITLE
Add alerts

### LINKED ISSUE #815: Add warning on buy and configure market widget about resolution dates
Buy widget:
Idle

configure market:
configure-market

Create a pdf out of the validity rules document:
https://docs.google.com/document/d/15NN5YUG6XLF3tgqGqWI8P7F9Hqxw2WDDfk7_UDW28EA/edit#heading=h.fhkxt5p147pu

**Figma:**[
Purchase Shares](https://www.figma.com/file/8ewqNUtmCOryLQDZiP6Rfc/V1?node-id=211%3A19995)
[Configure Market](https://www.figma.com/file/8ewqNUtmCOryLQDZiP6Rfc/V1?node-id=211%3A20358)

### PR DESCRIPTION
**Pull request by pimato**
Closes #815

### CODE REVIEW

**Comment by gabitoesmiapodo** [comment]
@pimato What's this PR for? Is there a related issue?

**Comment by pimato** [comment]
yeah there is: https://github.com/protofire/gnosis-conditional-exchange/issues/815

I´ve added an alert message for the buy interface:
Screenshot 2020-06-17 at 15 21 51

configure market interface:
Screenshot 2020-06-17 at 15 21 18

fund market interface:
Screenshot 2020-06-17 at 15 21 34


those warnings need to hyperlink to a new pdf document which I´ve also included.

A small change was also made configure market section where we increase the number of max outcomes to 8 instead of just 2.

**Comment by gabitoesmiapodo** [comment]
@pimato OK, good, I'll edit the first message so it closes automatically when we merge this.

File: app/src/components/market/sections/market_create/steps/items/funding_and_fee_step.tsx [hunk 1]
```diff
@@ -231,8 +237,7 @@ const FundingAndFeeStep = (props: Props) => {
             />
           </CurrenciesWrapper>
         )}
-        <WarningMessage />
-        <GridTransactionDetailsStyled noMarginTop={tokensAmount === 1}>
+        <GridTransactionDetailsStyled noMarginTop={false}>
```
Comment (gabitoesmiapodo) [inline app/src/components/market/sections/market_create/steps/items/funding_and_fee_step.tsx:240]:
```suggestion
        <GridTransactionDetailsStyled noMarginTop={false}>
```

You could make this prop optional if you find it troublesome. Also you could remove it if it isn't being used anymore.

File: app/src/components/market/sections/market_pooling/market_pool_liquidity.tsx [hunk 2]
```diff
@@ -63,6 +63,10 @@ const TabsGrid = styled.div`
   grid-template-columns: 1fr 1fr;
   margin: 0 0 25px;
 `
+const WarningMessageStyled = styled(WarningMessage)`
+  margin-top: 20px;
+  margin-bottom: 0px;
```
Comment (gabitoesmiapodo) [inline app/src/components/market/sections/market_pooling/market_pool_liquidity.tsx:68]:
```suggestion
  margin-bottom: 0;
```

File: app/src/components/market/sections/market_buy/market_buy.tsx [hunk 3]
```diff
@@ -194,6 +200,13 @@ const MarketBuyWrapper: React.FC<Props> = (props: Props) => {
           probabilities={probabilities}
           showPriceChange={amount.gt(0)}
         />
+        <WarningMessageStyled
+          description={
+            "Before trading on a market, make sure that its outcome will be known by its resolution date and it isn't an"
+          }
+          hyperlink={'./rules.pdf'}
```
Comment (gabitoesmiapodo) [inline app/src/components/market/sections/market_buy/market_buy.tsx:207]:
```
          hyperlink={'./rules.pdf'}
```

Maybe you can put this and `faq.pdf` in `app/src/common/constants.ts`. Will be handy if / when you have to change it.

Comment (pimato) [reply app/src/components/market/sections/market_buy/market_buy.tsx]: done!

**Pull request merged**